src/main/java/sparksoniq/jsoniq/runtime/iterator/RuntimeIterator.java

```diff
@@ -17,37 +17,53 @@
  * Author: Stefan Irimescu
  *
  */
- package sparksoniq.jsoniq.runtime.iterator;
+package sparksoniq.jsoniq.runtime.iterator;
```

Comment:
As a side note for future pull requests: it is best to put cosmetic/lint changes in separate pull requests on their own, it makes reviewing much easier as pull requests get smaller and have a distinct focus.